recipes/pyserini/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "0.11.0.0" %}
+
+package:
+  name: pyserini
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pyserini/pyserini-{{ version }}.tar.gz
+  sha256: 4ff93e607f570f0643321cb073aa571ee298a7eec20b21c4e13812e16a75a65b
+
+build:
+  # noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - openjdk >=11
+    - pip
+  run:
+    - python
+    - cython >=0.29.21
+    - faiss >=1.6.5
+    - importlib-metadata  # [py<38]
+    - numpy >=1.18.1
+    - pandas >=1.1.5
+    - pyjnius >=1.2.1
+    - pytorch
+    - scikit-learn >=0.22.1
+    - scipy >=1.4.1
+    # blocked on https://github.com/conda-forge/tensorflow-feedstock/pull/110
+    # - tensorflow >=2.3
```

Comment:
That PR for tensorflow has progressed very far now, and hopefully we'll have tensorflow builds on conda-forge in 1-2 weeks. Long-term, there are still some CI infra improvements necessary to consistently pytorch/tensorflow in conda-forge without too much manual intervention (currently running into the 6h azure timeout, not to mention not having GPUs in CI).

Reply:
@h-vetinari I don't think we need TensorFlow for pyserini. Let's just remove the dependency?

Reply:
Well, it's in `requirements.txt` [upstream](https://github.com/castorini/pyserini/blob/pyserini-0.11.0.0/requirements.txt), so I wouldn't know if that dependency is somehow optional... 😅

Reply:
It is not required. I'll double-check and remove it soon.

Reply:
removed

Reply:
OK, cool.

By the same token, does `pytorch` need to be added to `requirements.txt`? At least you mentioned it in your recipe [suggestion](https://github.com/castorini/pyserini/pull/426#issuecomment-814470309).

Note, it's also possible to build several different outputs (e.g. `pyserini-core` vs. `pyserini-tensorflow` vs. `pyserini-pytorch` vs. `pyserini-all`), in case that's interesting to you. Generally, packages should have the minimum amount of required dependencies, but if there's a good argument (usually optional code-path in the package) for certain dependencies, it can make sense to add more variants (pip/PyPI also support this with the optional `package[extras]` syntax).

Reply:
I feel we dont have to add pytorch to requirements. pytorch is usually installed by following guidance on official website I think?
But for conda forge we can try to contain everything